packages/discord.js/src/structures/ThreadChannel.js

```diff
@@ -196,6 +207,24 @@ class ThreadChannel extends Channel {
     return this.archiveTimestamp && new Date(this.archiveTimestamp);
   }
 
+  /**
+   * The time the thread was created at
+   * @type {?Date}
+   * @readonly
+   */
+  get createdAt() {
+    if (this.createdTimestamp !== null) {
+      return new Date(this.createdTimestamp);
+    }
+
+    // Private threads can rely on their ID for creation date
```

Comment:
Small nit for consistency!
```suggestion
    // Private threads can rely on their id for creation date
```